nemo/collections/common/tokenizers/text_to_speech/ipa_lexicon.py

```diff
@@ -184,6 +208,32 @@ def get_ipa_punctuation_list(locale):
                 '—',  # em dash, U+2014, decimal 8212
             ]
         )
+    if locale == "vi-VN":
+        punct_set.update(
```

Comment:
could you pls add the source of punctuations of Vietnamese?

Reply:
Hmm, it seems that there isn't any 'official' source that talks about Vietnamese punctuation marks. I can find some information about punctuation marks here: https://languagedrops.com/word/en/english/vietnamese/topics/punctuation/. 
Maybe we just need to use DEFAULT_PUNCTUATION.